src/libs/TransactionUtils.js

```diff
@@ -274,28 +274,30 @@ function getAllReportTransactions(reportID) {
  */
 function validateWaypoints(waypoints) {
     const waypointValues = _.values(waypoints);
-
     // Ensure the number of waypoints is between 2 and 25
     if (waypointValues.length < 2 || waypointValues.length > 25) {
         return false;
```

Comment:
return type mismatch. return `{}` instead of `false`
Also update jsDoc accordingly

Reply:
Still returns boolean.
This was my suggestion:
```diff
-       return false;
+       return {};
```